parsel/selector.py

```diff
@@ -35,12 +36,24 @@ def _st(st):
         raise ValueError('Invalid type: %s' % st)
 
 
-def create_root_node(text, parser_cls, base_url=None):
+def create_root_node(text, parser_cls, base_url=None, huge_tree=True):
     """Create root node for text using given parser class.
     """
+    lxml_version = parse_version(etree.__version__)
+    lxml_huge_tree_version = parse_version("4.2")
```

Comment:
I think version check (`lxml_version >= lxml_huge_tree_version`) should be done once, at module import - the result can be saved to a constant like LXML_SUPPORTS_HUGE_TREE

Reply:
Thanks for your input, I'll change it.